build-tools/packages/build-cli/docs/release.md

```diff
@@ -52,55 +53,99 @@ Generates a report of Fluid Framework releases.
+
+  -s, --highest
+      Always pick the greatest semver version as the latest (ignore dates).
+
+  -v, --verbose
+      Verbose logging.
 
 GLOBAL FLAGS
   --json  Format output as json.
 
 DESCRIPTION
   Generates a report of Fluid Framework releases.
 
-  The release report command is used to produce a report of all the packages that were released and their current
-  version. After a release, it is useful to generate this report to provide to customers, so they can update their
-  dependencies to the most recent version.
+  The release report command is used to produce a report of all the packages that were released and their version. After
+  a release, it is useful to generate this report to provide to customers, so they can update their dependencies to the
+  most recent version.
 
-  The command will prompt you to select versions for a package or release group in the event that multiple versions have
-  recently been released.
+  The command operates in two modes: "whole repo" or "release group." The default mode is "whole repo." In this mode,
+  the command will look at the git tags in the repo to determine the versions, and will include all release groups and
+  packages in the repo. You can control which version of each package and release group is included in the report using
+  the --interactive, --mostRecent, and --highest flags.
 
-  Using the --all flag, you can list all the releases for a given release group or package.
+  The "release group" mode can be activated by passing a --releaseGroup flag. In this mode, the specified release
+  group's version will be loaded from the repo, and its immediate Fluid dependencies will be included in the report.
+  This is useful when we want to include only the dependency versions that the release group depends on in the report.
 
 EXAMPLES
   Output all release report files to the current directory.
 
-    $ flub release report -o .
+    $ flub release report
 
-  Generate a minimal release report and display it in the terminal.
+  Generate a release report of the highest semver release for each package and release group and display it in the
+  terminal only.
 
-    $ flub release report
+    $ flub release report --no-output
+
+  Generate a release report for each package and release group in the repo interactively.
 
-  Generate a minimal release report and output it to stdout as JSON.
+    $ flub release report -i
+```
+
+## `flub release report all`
 
-    $ flub release report --json
+Generates a report of all releases of a particular package or release group.
+
+```
+USAGE
+  $ flub release report all [-g client|server|azure|build-tools | -p <value>] [-l <value>] [-v]
 
+FLAGS
+  -g, --releaseGroup=<option>  Name of the release group
+                               <options: client|server|azure|build-tools>
+  -l, --limit=<value>          Limits the number of displayed releases for each release group.
```

Comment:
I don't see anything about the ordering of the report, so a limit is confusing. Do we include the first `l` packages alphabetically? The most recent `l` historical versions (by date or version number?).

Reply:
It's ordered by semver. I'll clarify the text.